mlflow/gateway/app.py

```diff
@@ -113,7 +113,7 @@ async def health():
     @app.get("/gateway/routes/{route_name}")
     async def get_route(route_name: str):
         if matched := app.get_dynamic_route(route_name):
-            return {"route": matched}
+            return matched
```

Comment:
Should we also fix the search-routes response structure?

Reply:
Looks like we can't becasuse we might add the token for pagination

Reply:
Right :). SearchRoutes should return a payload with a `routes` key :)